app/views/govuk_publishing_components/components/_radio.html.erb

```diff
@@ -122,7 +122,7 @@
           <% end %>
 
           <% if item[:conditional] %>
-            <%= tag.div item[:conditional], class:"govuk-radios__conditional govuk-radios__conditional--hidden", id: conditional_id %>
+            <%= tag.div item[:conditional], class:"govuk-radios__conditional govuk-radios__conditional--hidden govuk-body", id: conditional_id %>
```

Comment:
I'd be hesitant to change this as presumably wherever this is used the content passed to the component is already styled by the tags and classes inside it - I don't know what effect wrapping that in a `govuk-body` class would have.

Reply:
🤔  Good point, let me check some more instances

Reply:
Decided to drop this commit. I need to properly test with all the other instances where this appears across the site but _finding_ them is out of scope of this. Will capture.